var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -52,7 +52,7 @@ class Hdf5(Package):
     variant('cxx', default=True, description='Enable C++ support')
     variant('fortran', default=True, description='Enable Fortran support')
 
-    variant('mpi', default=False, description='Enable MPI support')
+    variant('mpi', default=True, description='Enable MPI support')
```

Comment:
I could. But shouldn't all packages have mpi default to true? Spack _is_ a Supercomputer PACKage manager after all.